### DISCUSSION TITLE
关于宣传

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#34] [Announcements]
> ~~当然这条东西也不是我写的，我的文笔没那么好~~ （不必复制这句）

为了更好地帮助各位考生，我们发起了一个项目，意在收集全国各高校招生时不会写明，却会实实在在影响大学生活质量的要求与细节。

如果您是考生或考生家长，您可以点击 https://colleges.chat 或 https://github.com/CollegesChat/university-information/discussions/51 查看我们已收集到的信息。
GitHub 备用镜像：https://hub.fastgit.org/CollegesChat/university-information/discussions/51

如果您是在读大学生，您可以通过这个问卷 https://www.wenjuan.com/s/UZBZJv8lWrd/ 匿名提交一份宝贵的数据，我们将在确认真实性后将其添加到信息库中。

如果您是教职工，也欢迎您通过以上链接提交您所在学校的信息，或查阅已有信息并借此改善您所在学校的生活质量。

同时，为了进一步扩大数据来源，我们欢迎您以任何形式（包括复制后粘贴）转发此条消息，您的每一次转发都是莫大的帮助。感谢！

**Comment by renbaoshuo** [comment]
这是一条测试回复，用来测试IFTTT配置是否正确。

**Reply by YanWQ-monad** [reply]
再测试一下 🤔

**Reply by renbaoshuo** [reply]
这是第二条测试消息。

**Reply by YanWQ-monad** [reply]
这是第三条测试消息。
